### PR TITLE
OPSEXP-1649 Fix uninstall playbook failure at clustered EC2 test

### PR DESCRIPTION
**Pull request by upandey123**
Fixed cleanup.yml and platform-uninstall.yml files

### CODE REVIEW

File: molecule/multimachine/cleanup.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 - name: Default cleanup
-  import_playbook: ../default/cleanup.yml
+  import_playbook: ../default/cleanup-uninstall.yml
```
Comment (alxgomz) [inline molecule/multimachine/cleanup.yml:3]: This must be wrong given the results of the `multimachine` build stage

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: molecule/multimachine/cleanup.yml [hunk 2]
```diff
@@ -30,24 +30,28 @@
...
+    broker_name: molecule-{{ lookup('ansible.builtin.env', 'BRANCH_NAME')[:42] }}-{{ lookup('ansible.builtin.env', 'BUILD_NUMBER') }}
+    
   tasks:
 
     - name: Grab activemq broker id
       ansible.builtin.shell: |
         aws mq describe-broker --broker-id {{ broker_name }} || true
       register: describe_broker_output
 
+
```
Comment (alxgomz) [inline molecule/multimachine/cleanup.yml:42]: remove double blank lines pls

File: molecule/multimachine/prepare.yml [hunk 3]
```diff
@@ -84,7 +84,8 @@
   hosts: localhost
   gather_facts: false
   vars:
-    broker_name: molecule-{{ lookup('env', 'BUILD_NUMBER') }}
+    #broker_name: molecule-{{ lookup('env', 'BUILD_NUMBER') }}
```
Comment (alxgomz) [inline molecule/multimachine/prepare.yml:87]: Remove this comment pls

**Review by alxgomz — CHANGES_REQUESTED** [review]
Your branch as some conflict with the merge target now. Please rebase from master.

**Review by alxgomz — APPROVED** [review]

File: molecule/multimachine/cleanup.yml [hunk 4]
```diff
@@ -42,7 +42,7 @@
       set_fact:
         broker_id: "{{ (describe_broker_output.stdout | from_json).BrokerId }}"
       when: describe_broker_output.stdout | length > 0
-
+
```
Comment (alxgomz) [inline molecule/multimachine/cleanup.yml:45]: this one remains

File: molecule/multimachine/cleanup.yml [hunk 5]
```diff
@@ -30,7 +30,7 @@
   hosts: localhost
   gather_facts: false
   vars:
-    broker_name: molecule-{{ lookup('ansible.builtin.env', 'BRANCH_NAME')[:42] }}-{{ lookup('ansible.builtin.env', 'BUILD_NUMBER') }}
+    broker_name: molecule-{{ lookup('ansible.builtin.env', 'BRANCH_NAME')[:42] }}-{{ lookup('ansible.builtin.env', 'BUILD_NUMBER') }}
```
Comment (alxgomz) [inline molecule/multimachine/cleanup.yml:33]: This breaks the pre-commit checks

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: molecule/multimachine/cleanup.yml (hunk 5 above)
Comment (upandey123) [reply molecule/multimachine/cleanup.yml]: fixed the pre commit checks.

File: molecule/multimachine/cleanup.yml (hunk 4 above)
Comment (upandey123) [reply molecule/multimachine/cleanup.yml]: fixed that one too.

**Review by gionn — APPROVED** [review]

**Pull request merged**